rollup.types.config.ts

```diff
@@ -25,6 +25,33 @@ export default rollup.defineConfig({
     return externals.some(external => source.startsWith(external))
   },
   plugins: [
+    {
+      // Remove "declare module 'vscode'" from vscode type
+      // Or else dts plugin is unable to extract types from it
+      name: 'remove-vscode-declare-module',
```

Comment:
nit: in order to keep the rollup config easy to read, I think you should extract that code into a separate function/plugin.

Reply:
be2c9c6ed7047fc5e1958c4a16d18ff83b6fb805